code/LINDA/LINDA_fire.dm

```diff
@@ -131,7 +143,7 @@
 				if(!(location.atmos_adjacent_turfs & direction))
 					continue
 				var/turf/simulated/T = get_step(src, direction)
-				if(istype(T) && T.active_hotspot)
+				if(istype(T) && !T.active_hotspot)
```

Comment:
While I can't explain why this is a thing, it's identical to -tg-'s procs, I would rather it is not touched. Perhaps ask @Aranclanos what this is actually supposed to do.